blog/BLOG.md

```diff
@@ -50,24 +50,31 @@ Cerberus is a collection of the following tasks that work together to achieve a
 
 ## Where to go from here?
 
-Cerberus is compatible with any project that uses [fastlane] or [gradle] and [Jira] as the project tracker. If you would like to give it a spin:
+Cerberus is compatible with any project that uses [Fastlane] or [Gradle] and [Jira] as the project tracker. Here are some links if you would like to give it a spin:
 
-! Add pending links and instructions here!
+**Fastlane**
 
+- [Project Page](https://github.com/outware/fastlane-plugin-cerberus)
+- [Integration Instructions](https://github.com/outware/fastlane-plugin-cerberus/blob/develop/INTEGRATION.md)
```

Comment:
Is it worth while linking to the integration instructions from the readme as well?

Reply:
Added in d5df4d8